### PR TITLE
Eda in r studio filtering grouping plotting data

### PR DESCRIPTION
**Pull request by Sara-Eftekhar-Azam**

### CODE REVIEW

**Review by Phaneron97 — CHANGES_REQUESTED** [review]
It's unclear what has been done in this PR.

**Comment by Sara-Eftekhar-Azam** [comment]
> It's unclear what has been done in this PR.

I have added some code that will import packages that are needed for the rest of the code such as ggplot2.
Next, I import mpg dataset from ggplot2 package. 
I have created a number of plots to illustrate different aspects of this dataset, including filtering data and grouping data.

**Review by Bastiaan27 — APPROVED** [review]
I really like this! The only thing to add is to create small and concise exsercises they can try on their own but try to keep it short so we don't run short on time 😄

**Comment by Sara-Eftekhar-Azam** [comment]
I update the EDA part. I added 3 assignments in the file for participants to complete. They are very similar to what I have explained in the file.

**Comment by Sara-Eftekhar-Azam** [comment]
> I really like this! The only thing to add is to create small and concise exsercises they can try on their own but try to keep it short so we don't run short on time 😄

Thanks for the comment. I added some short exercises (I called them assignments) in the code. Please have a look at them too.

File: code/EDA.Rmd [hunk 1]
```diff
@@ -110,90 +88,52 @@ Filtering in R is like adding conditions with WHERE in SQL.
 Plotting is rather simple and intuitive. 
 
 Let's visualize the city and highway mileage (mile per gallon of fuel).
-The ggplot library acts in an additive way. You can add more and amore to the figure. 
-Task: plot city vs highway mileage. Show engine size and vehicle class as size and color.
```
Comment (Phaneron97) [inline code/EDA.Rmd]: I thought we spoke about using the same dataset? This probably makes it more difficult for us and the participants to check the correctness of all tasks and code. We should probably stick to just 1 dataset (the one in folder `data/` and do all analysis tasks on that.

**Review by Phaneron97 — CHANGES_REQUESTED** [review]
The tasks are well put together imo, great start although we should stick to just 1 dataset to make things easier for us and the participants. Could you use the dataset in `data/` during the tasks instead?

**Review by Bastiaan27 — APPROVED** [review]
Nice work!!

**Review by Phaneron97 — APPROVED** [review]
all good, please merge 👍

**Pull request merged**